content/doc/book/updating/windows.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: section
+title: Windows
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:left
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Windows
+
+There are two ways to upgrade Jenkins in Windows.
+One is via Windows MSI installer
+Other one is discussed later on the page.
+
+== Upgrading via MSI Installer
+you can download the latest LTS or Weekly MSI Installer to upgrade Jenkins.
+
+Just Download the Jenkins 
+link:https://www.jenkins.io/download/#downloading-jenkins[MSI Installer] from here and install the latest
+Jenkins LTS or Weekly.
+for more information visit this page.
+link:https://www.jenkins.io/doc/book/installing/windows/#windows-msi-installers[Installing Jenkins On Windows via MSI Installer]
+
+=== 1. Stop the running Jenkins service
+
+To stop Jenkins Please avoid shutting down the Java process or the Windows service. These are not usual commands. Use those only if your Jenkins is causing problems.
+
+use the following command to stop server
+`http://[jenkins-server]/exit`
+
+Example: if my local PC is running Jenkins at port 8080, it will be
+`http://localhost:8080/exit`
+
+=== 2. Go to the location where your Jenkins instance is installed
+	
+By Default Jenkins is installed here: 
+`C:\Program Files\Jenkins`
+
+=== 3. Rename the war folder to war.old
+This is important if the upgrade process fails 
+you won't lose the backup of your Jenkins instance
+
+=== 4. Rename jenkins.war file to jenkins.war.old
+This is important for backup so *Don't Replace and Don't Delete* the old war file.
+
+=== 5. Copy the new jenkins.war you downloaded into your Jenkins installation directory
+This is the upgraded jenkins.war which will update your Jenkins Instance
+
+=== 6. Start Jenkins service
+
+To start the service you need to open the terminal in the directory where your Java WAR File is present and run the following command
+[source]
+```
+java -jar jenkins.war
+```
+
+=== 7. Observe the creation of a new war directory which will now contain the contents of the new jenkins.war archive
+
+By doing so, you can avoid losing your setup and builds that happen when a WAR file is replaced.
```

Comment:
```suggestion
=== Observe the creation of a new war directory which will now contain the contents of the new jenkins.war archive

By doing so, you can avoid losing your setup and builds that happen when a WAR file is replaced.
```
Is this its own step or could it be added to the previous step as a result, or just a separate result that is not part of the steps?